pkg/apis/ceph.rook.io/v1/resources.go

```diff
@@ -27,8 +27,14 @@ const (
 	ResourcesKeyMgr = "mgr"
 	// ResourcesKeyMgrSidecar represents the name of resource in the CR for a mgr
 	ResourcesKeyMgrSidecar = "mgr-sidecar"
-	// ResourcesKeyOSD represents the name of resource in the CR for an osd
+	// ResourcesKeyOSD represents the name of resource in the CR for an all osd
 	ResourcesKeyOSD = "osd"
+	// ResourcesKeyHDDDeviceOSD represents the name of resource in the CR for an osd with HDD device
+	ResourcesKeyHDDDeviceOSD = "osd-hdd"
```

Comment:
Similar to other comments, I don't believe we need these specific consts for device classes.